<?xml version='1.0' ?>
<launch>

  
  <include file="$(find-pkg-share barista_dock_charger_station)/launch/barista_1_charge_server_v3.launch.xml">
    <arg name="robot_name" value="turtleE_1"/>
  </include>


</launch>